Bluejay/Bluejay/Bluejay.swift

```diff
@@ -558,6 +563,9 @@ extension Bluejay: CBCentralManagerDelegate {
         UIApplication.shared.endBackgroundTask(backgroundTask)
     }
     
+    /**
+     Examine the listen cache in `UserDefaults` to determine whether there are any listens that might need restoration.
```

Comment:
@JeremyChiang @nbrooke Do we want to put this much implementation detail into the docs? Typically I'd advocate for us talking about the inputs, outputs, and any side effects developers should know about, rather than how we implement the method.